modules/pafRtdProvider.md

```diff
@@ -0,0 +1,93 @@
+## Prebid Addressability Framework Real-time Data Submodule
+
+The PAF real-time data module in Prebid has been built so that publishers
+can quickly and easily setup the Prebid Addressability Framework and utilize OneKey.
+This module is used along with the pafIdSysytem to pass PAF data to your partners.
+Both modules are required. This module will pass transmission requests to your partners
+while the pafIdSystem will pass the pafData.
+
+Background information:
+https://github.com/prebid/addressability-framework
+https://github.com/prebid/paf-mvp-implementation
+
+
+
+### Publisher Usage
+
+The paf RTD module depends on paf-lib.js existing in the page.
+
+Compile the paf RTD module into your Prebid build:
+
+`gulp build --modules=userId,pafIdSystem,rtdModule,pafRtdProvider,appnexusBidAdapter`
+
+Add the PAF RTD provider to your Prebid config. In this example we will configure
+a sample proxyHostName. See the "Parameter Descriptions" below for more detailed information
+of the configuration parameters.
+
+```
+pbjs.setConfig(
+    ...
+    realTimeData: {
+        auctionDelay: 5000,
+        dataProviders: [
+            {
+                name: "paf",
+                waitForIt: true,
+                params: {
+                    proxyHostName: "cmp.pafdemopublisher.com"
+                }
+            }
+        ]
+    }
+    ...
+}
+```
+
+### Parameter Descriptions for the PAF Configuration Section
+
+| Name  |Type | Description   | Notes  |
+| :------------ | :------------ | :------------ |:------------ |
+| name | String | Real time data module name | Always 'paf' |
+| waitForIt | Boolean | Required to ensure that the auction is delayed until prefetch is complete | Optional. Defaults to false |
+| params | Object | | |
+| params.proxyHostName | String | URL of the PAF Proxy which will generate seeds. | Required |
```

Comment:
This is not a URL but just a servername. Maybe you could provide an example as well to avoid confusion.

Reply:
changed it to servername, the example is above the table